### PR TITLE
Feature/npg 3367 test delegator endpoint

### PR DESCRIPTION
**Pull request by kukkok3**

### CODE REVIEW

File: src/vit-servicing-station/vit-servicing-station-tests/src/tests/snapshots/snapshot.rs [hunk 1]
```diff
@@ -91,9 +149,11 @@ pub fn replace_snapshot_with_tag() {
     let second_snapshot = SnapshotBuilder::default()
         .with_timestamp(first_snapshot.content.update_timestamp + 1)
         .build();
-
     rest_client.put_snapshot_info(&second_snapshot).unwrap();
+
     for (idx, entry) in first_snapshot.content.snapshot.iter().enumerate() {
+        println!("{:#?}", entry);
```
Comment (dkijania) [inline src/vit-servicing-station/vit-servicing-station-tests/src/tests/snapshots/snapshot.rs:155]: debug printing?

Comment (kukkok3) [reply src/vit-servicing-station/vit-servicing-station-tests/src/tests/snapshots/snapshot.rs]: fixed

File: src/vit-servicing-station/vit-servicing-station-tests/src/common/snapshot.rs [hunk 2]
```diff
@@ -124,6 +125,20 @@ pub struct VoterInfo {
     pub voter_info: Vec<VotingPower>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct DelegatorInfo {
```
Comment (Mr-Leshiy) [inline src/vit-servicing-station/vit-servicing-station-tests/src/common/snapshot.rs:129]: Why not use already defined `DelegatorInfo` struct from https://github.com/input-output-hk/catalyst-core/blob/8071368992ae04aa56f9ce15ccb144c0fdee2e6b/src/vit-servicing-station/vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs#L106 ?

Comment (kukkok3) [reply src/vit-servicing-station/vit-servicing-station-tests/src/common/snapshot.rs]: right:)

**Review by nicopado — APPROVED** [review]

**Pull request merged**